src/uct/ib/ud/base/ud_iface.c

```diff
@@ -1063,6 +1065,10 @@ void uct_ud_iface_vfs_refresh(uct_iface_h iface)
     ucs_vfs_obj_add_ro_file(ud_iface, ucs_vfs_show_primitive,
                             &ud_iface->config.tx_qp_len, UCS_VFS_TYPE_INT,
                             "tx_qp_len");
+
+    ucs_ptr_array_for_each(ep, i, &ud_iface->eps) {
+        uct_ud_ep_vfs_populate(ep);
```

Comment:
Please add `ucs_vfs_obj_remove` for each UD endpoint when it is destroyed.

Reply:
i think it's already done here:
https://github.com/yosefe/ucx/blob/8d014c0c7a0b6d2fdbddaa25e697828d4d02058c/src/uct/base/uct_iface.c#L607

Reply:
Yes, I see.